nparse.py

```diff
@@ -1,205 +1,11 @@
 """NomnsParse: Parsing tools for Project 1999."""
 import os
 import sys
-import webbrowser
 
-from PySide6.QtCore import Qt
-from PySide6.QtGui import QCursor, QFontDatabase, QIcon
-from PySide6.QtWidgets import QApplication, QFileDialog, QMenu, QSystemTrayIcon
-import semver
-
-from helpers import config, logreader, resource_path, get_version
-from helpers.settings import SettingsWindow, SettingsSignals
-from helpers.logreader import LogReaderSignals
-from helpers.location_service import LocationSharingService, LocationSharingSignals
-import parsers
-from parsers.maps.window import MapsSignals
-
-try:
```

Comment:
I think everything from 18-39 should possibly stay in here? It's all very clearly "run once on init" stuff and this file will still be the primary app entry point, right?

Reply:
Good call, updated and moved the pyinstaller try block back to the main nparse.py file